packages/zwave-js/src/lib/controller/FirmwareUpdateService.ts

```diff
@@ -124,7 +124,9 @@ export function getAvailableFirmwareUpdates(
 	productId: number,
 	firmwareVersion: string,
 	apiKey?: string,
+	userAgent?: string,
```

Comment:
@martinhjelmare @raman325

Let's add this to Home Assistant so we can at the firmware service the requests from HA

Reply:
I'll add a way to globally configure the user agent on the driver instance later.